Cargo.lock

```diff
@@ -938,31 +938,45 @@ dependencies = [
 
 [[package]]
 name = "cuid"
-version = "1.3.2"
-source = "git+https://github.com/prisma/cuid-rust?branch=wasm32-support#ccfd958c224c79758c2527a0bca9efcd71790a19"
+version = "1.3.3"
+source = "git+https://github.com/prisma/cuid-rust?branch=v1.3.3-wasm32-unknown-unknown#dc68c4f47a3dbcd511f605135ac7c948775a3ab9"
```

Comment:
anything to contribute upstream?

Reply:
Yes, but let's do that after we're out with Prisma 6